test/linting/lint.sh

```diff
@@ -1,4 +1,4 @@
-#! /bin/bash
+#!/usr/bin/env bash
```

Comment:
IMO this is the more common syntax. However all of our scripts have a space before the path, so we should do the same here..

```suggestion
#! /usr/bin/env bash
```